web/ui_utils.js

```diff
@@ -862,6 +862,18 @@ function toggleExpandedBtn(button, toggle, view = null) {
   view?.classList.toggle("hidden", !toggle);
 }
 
+// In Firefox, the css calc function uses f32 precision but the Chrome or Safari
+// are using f64 one. So in order to have the same rendering in all browsers, we
+// need to use the right precision in order to have correct dimensions.
+const calcRound = (function () {
+  if (typeof document === "undefined") {
```

Comment:
Given that this should only be needed in order for the unit-tests to run successfully in Node.js, let's copy the approach used in `animationStarted` above.
```suggestion
  if (
    typeof PDFJSDev !== "undefined" &&
    PDFJSDev.test("LIB") &&
    typeof document === "undefined"
  ) {
```